uaa/src/test/java/org/cloudfoundry/identity/uaa/integration/feature/LoginIT.java

```diff
@@ -531,33 +531,65 @@ private void loginThroughDiscovery(String userEmail, String password) {
     }
 
     @Test
-    public void testSelfServiceLinksBehavior() {
+    public void testSelfServiceResetPasswordLinksBehavior() {
         RestTemplate adminClient = IntegrationTestUtils.getClientCredentialsTemplate(
-                IntegrationTestUtils.getClientCredentialsResource(baseUrl, new String[0], "admin", "adminsecret"));
+            IntegrationTestUtils.getClientCredentialsResource(baseUrl, new String[0], "admin", "adminsecret"));
         String zoneId = "testzone3";
         String zoneUrl = baseUrl.replace("localhost", zoneId+".localhost");
         IdentityZone testZone3 = IntegrationTestUtils.createZoneOrUpdateSubdomain(adminClient, baseUrl, zoneId, zoneId, new IdentityZoneConfiguration());
 
-        testZone3.getConfig().getLinks().setSelfService(new Links.SelfService().setSelfServiceLinksEnabled(true).setPasswd("").setSignup(""));
+        testZone3.getConfig().getLinks().setSelfService(new Links.SelfService().setSelfServiceResetPasswordEnabled(true).setPasswd(""));
```

Comment:
Minor.  Each test case below calls `new Links.SelfService()...`.  You could initialize it once at the beginning of the test and then re-use.

Reply:
Done